docs/source/_ext/garak_ext.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+
+from docutils.frontend import OptionParser
+from docutils.utils import new_document
+from sphinx.util.docutils import SphinxDirective
+from sphinx.parsers import RSTParser
+
+
+class ShowASRDirective(SphinxDirective):
+    has_content = True
+
+    def run(self) -> list:
+        rst = ""
+        with open("calibration.json", encoding="utf-8") as f:
+            calibration = json.load(f)
+            for key in calibration:
+                if key.startswith(self.env.docname.replace("garak.probes.", "")):
+                    probe, detector = key.split("/")
+                    scores = calibration[key]
+                    probe_ref = f":obj:`~garak.probes.{probe}`"
+                    detector_ref = f":obj:`~garak.detectors.{detector}`"
+
+                    rst += f"\n* {probe_ref}: {100*scores["mu"]:.1f}% (w/ detector {detector_ref})"
+
+        if rst:
+            rst = (
+                """\nAttacks with this probe have the following attack success rates (ASR) in a recent `evaluation <https://github.com/NVIDIA/garak/blob/main/garak/data/calibration/bag.md>`_:\n"""
```

Comment:
Separate concern, this comment is placed at the `module` level consider using plurals to denote the probe classes each have a rate and avoid confusion where users may conflate `probe family` and `individual probes`:

```suggestion
                """\nAttacks with these probes have the following attack success rates (ASR) in a recent `evaluation <https://github.com/NVIDIA/garak/blob/main/garak/data/calibration/bag.md>`_:\n"""
```

For example the `dan` family has 13 probes of which 3 have calibration 2 of which have multiple detectors.

<img width="736" height="376" alt="Image" src="https://github.com/user-attachments/assets/7309a73a-998b-4931-9b74-d2009ca4fc15" />

Reply:
Lingo questions:

- Have we been using _attack_ as a term?  Is there a synonym that has been in prevalent use?
- Could the link text be "calibration" (if that is true...)?

nit: spell out with